apps/src/code-studio/components/ExportAllowedDialog.jsx

```diff
@@ -0,0 +1,744 @@
+/* global dashboard */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+import BaseDialog from '../../templates/BaseDialog';
+import AbuseError from './AbuseError';
+import color from '../../util/color';
+import {hideExportDialog} from './exportDialogRedux';
+import i18n from '@cdo/locale';
+import firehoseClient from '../../lib/util/firehose';
+import exportExpoIconPng from '../../templates/export/expo/icon.png';
+import SendToPhone from './SendToPhone';
+
+function recordExport(type) {
+  if (!window.dashboard) {
+    return;
+  }
+
+  firehoseClient.putRecord(
+    {
+      study: 'finish-dialog-export',
+      study_group: 'v1',
+      event: 'project-export',
+      project_id: dashboard.project && dashboard.project.getCurrentId(),
```

Comment:
I think you can import `project` directly without referring to the dashboard global, and it'll work just as well.